numpy/core/src/umath/override.c

```diff
@@ -563,18 +562,24 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
                     Py_DECREF(errmsg);
                 }
             }
+            Py_DECREF(override_args);
             goto fail;
         }
 
         /*
          * Set the self argument of our unbound method.
          * This also steals the reference, so no need to DECREF after.
          */
-        PyTuple_SetItem(override_args, 0, override_obj);
+        if (PyTuple_SetItem(override_args, 0, override_obj) < 0) {
```

Comment:
Can this call fail? I'm actually not entirely sure why we weren't using `SET_ITEM` here -- we really do know things will be OK here, no?